pom.xml

```diff
@@ -128,6 +129,18 @@
                 <version>${version.com.google.guava.guava}</version>
             </dependency>
             <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpclient</artifactId>
+                <scope>provided</scope>
+                <version>${version.org.apache.httpcomponents.httpclient}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpcore</artifactId>
+                <version>${version.org.apache.httpcomponents.httpcore}</version>
+                <scope>provided</scope>
+            </dependency>
```

Comment:
It is, bud provide scope is not transitive. [maven transitive](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html)